service/src/main/java/com/zebrunner/reporting/service/feign/FeignConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package com.zebrunner.reporting.service.feign;
+
+import feign.Request;
+import feign.Retryer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+public class FeignConfiguration {
+
+  @Bean
+  public Request.Options timeoutConfiguration(){
+    return new Request.Options(5000, TimeUnit.MILLISECONDS,
```

Comment:
This line is longer that 120 chars? Just wondering

Reply:
Happened to be 93

Reply:
Got it, no line breaks here